rest-service/manager_rest/storage/management_models.py

```diff
@@ -616,6 +617,24 @@ class RabbitMQBroker(_WithCACert, SQLModelBase):
     # additional params, as **kwargs to creating the connection
     params = db.Column(JSONString, nullable=True)
     networks = db.Column(JSONString)
+    node_id = db.Column(db.Text, unique=True, nullable=False)
+    is_external = db.Column(db.Boolean, default=False, nullable=False)
+
+    @classmethod
+    def unique_id(cls):
```

Comment:
why do we need this func and the next one?

Reply:
Because we don't have `id` field and we inherit SQLModelBase.